src/utils/MemoryFS.js

```
@@ -0,0 +1,471 @@
+  async readFile(path, options) {
+    const p = normalizePath(path)
+    const entry = this._store.get(p)
+
+    if (!entry) {
+      const err = new Error(`ENOENT: no such file or directory, open '${p}'`)
+      err.code = 'ENOENT'
+      throw err
+    }
+
+    if (entry.type !== 'file') {
+      const err = new Error(`EISDIR: illegal operation on a directory, read '${p}'`)
```

ESLint

[error] Replace ``EISDIR:·illegal·operation·on·a·directory,·read·'${p}'`` with `⏎········`EISDIR:·illegal·operation·on·a·directory,·read·'${p}'`⏎······`

(prettier/prettier)